example/src/test/scala/monocle/function/AtExample.scala

```diff
@@ -34,4 +38,21 @@ class AtExample extends MonocleSuite {
     (Set(1, 2, 3) applyLens at(2) set None)     shouldEqual Set(1, 3)
   }
 
+  test("at creates a Lens from Int to one of its bit") {
+    (3 applyLens at(0: IntBits) get) shouldEqual true   // true  means bit is 1
```

Comment:
Is the `IntBits` ascription necessary? The `auto._` import should take care of converting `Int` to `IntBits`, right?